### PR TITLE
Show firstparty protections ui

### LINKED ISSUE #2482: Indicate when first-party protections are active
>Besides automatic tracker blocking, Privacy Badger removes outgoing link click tracking on [Facebook](https://www.eff.org/deeplinks/2018/05/privacy-badger-rolls-out-new-ways-fight-facebook-tracking) and [Google](https://www.eff.org/deeplinks/2018/10/privacy-badger-now-fights-more-sneaky-google-tracking), with more privacy protections on the way.

The thing is that we don't advertise that this happens anywhere in the UI. But we should! We should inform the user when Privacy Badger is actively removing tracking from outgoing links on Facebook and Google.

### PR DESCRIPTION
**Pull request by ablanathtanalba**
Fixes #2482 

This adds a new message to the badger popup when any of the first party scripts are being run on the user's current site. If in the future the first party scripts are doing things other than anchor tag scrubbing, then maybe we could change the messaging.... until then, this oughtta work!

### CODE REVIEW

**Comment by ablanathtanalba** [comment]
Oh, and the link that the new text points to is the deeplinks blog post that describes PB's link tracking protections: https://www.eff.org/deeplinks/2018/05/privacy-badger-rolls-out-new-ways-fight-facebook-tracking

**Comment by ablanathtanalba** [comment]
Rebased on master and added another commit to simplify messaging into one case for all variations of the popup, regardless of current detected tracker count.

File: src/skin/popup.html [hunk 1]
```diff
@@ -129,6 +129,7 @@ <h2 id="title-name"><span class="i18n_name"></span></h2>
       <span class="i18n_popup_instructions_no_trackers" data-i18n_contents_placeholders="<a target='_blank' title='i18n_what_is_a_tracker' class='tooltip' href='https://privacybadger.org/#What-is-a-third-party-tracker'>"></span>
       <span id="no-third-parties" class="i18n_popup_blocked" style="display:none"></span>
     </span>
+    <span id="instructions_firstparty_protections" class="i18n_popup_info_firstparty_protections" style="display:none"></span>
```
Comment (ghostwords) [inline src/skin/popup.html:132]: CSS names should use dash separators going forward: `instructions-firstparty-protections`.

File: src/js/popup.js [hunk 2]
```diff
@@ -599,7 +604,6 @@ function refreshPopup() {
 
     // show "no trackers" message
     $("#instructions-no-trackers").show();
-
```
Comment (ghostwords) [inline src/js/popup.js:602]: Why remove? Helps legibility, I think.

File: src/_locales/en_US/messages.json [hunk 3]
```diff
@@ -423,6 +423,10 @@
             }
         }
     },
+    "popup_info_firstparty_protections": {
+        "message": "Protection from <a href='https://www.eff.org/deeplinks/2018/05/privacy-badger-rolls-out-new-ways-fight-facebook-tracking'>link tracking</a> is enforced on this site",
```
Comment (ghostwords) [inline src/_locales/en_US/messages.json:427]: https://www.eff.org/deeplinks/2018/05/privacy-badger-rolls-out-new-ways-fight-facebook-tracking is specific to Facebook. Maybe link to https://privacybadger.org/#What-about-tracking-by-the-sites-I-actively-visit%2c-like-NYTimes.com-or-Facebook.com instead?

File: src/skin/popup.html (hunk 1 above)
Comment (ghostwords) [reply src/skin/popup.html:132]: Should probably show this section above the "X trackers blocked" section, so as not to come between it and the slider list.

**Review by ghostwords — COMMENTED** [review]
Thank you, got some feedback.

**Comment by ablanathtanalba** [comment]
Cool, thanks for the feedback. I hit on each of the smaller points and added some dynamic checking to cover all first party urls and future ones. Here's an updated img of how the popup looks:

Screen Shot 2021-01-28 at 12 49 00 PM

File: src/js/popup.js [hunk 4]
```diff
@@ -169,6 +169,11 @@ function init() {
     $overlay.toggleClass('active');
   }
 
+  // show firstparty link tracking protection if we scrub links on this tab host
+  if (POPUP_DATA.tabHost == 'www.facebook.com' || POPUP_DATA.tabHost == 'www.google.com') {
```
[the remark this answers is not in the document]
Comment (ablanathtanalba) [reply src/js/popup.js]: fixed this to pull down all domains from our manifest

**Comment by ablanathtanalba** [comment]
here's an updated pic for how this change looks in the popup:
Screen Shot 2021-02-24 at 8 01 39 PM

**Comment by ablanathtanalba** [comment]
This is gonna be wrapped up in #2748, so I'm closing this in favor of that one

**Pull request closed without merging**